### PR TITLE
Consistent labels

### PR DESCRIPTION
**Pull request by ericfell**

### CODE REVIEW

File: src/rfbzero/redox_flow_cell.py [hunk 1]
```diff
@@ -153,17 +157,18 @@ def __init__(
         if not 0.0 < self.alpha_cls < 1.0 or not 0.0 < self.alpha_ncls < 1.0:
             raise ValueError("Alpha parameters must be between 0.0 and 1.0")
 
-        if not isinstance(self.n_cls, int) or not isinstance(self.n_ncls, int):
-            raise ValueError("'n_cls' and 'n_ncls' must be integers")
+        if not isinstance(self.num_electrons_cls, int) or not isinstance(self.num_electrons_ncls, int):
+            raise ValueError("'num_electrons_cls' and 'num_electrons_ncls' must be >= 1")
```
Comment (jeremyfell) [inline src/rfbzero/redox_flow_cell.py:161]: The check and the error message don't match. Let's remove the isinstance check, just ensure >= 1, and split these into two checks:
```py
if self.num_electrons_cls < 1:
    raise ValueError("'num_electrons_cls' must be >= 1")

if self.num_electrons_ncls < 1:
    raise ValueError("'num_electrons_ncls' must be >= 1")
```

File: src/rfbzero/redox_flow_cell.py [hunk 2]
```diff
@@ -391,17 +396,17 @@ def _coulomb_counter(
 
         Returns
         -------
-        delta_ox : float
-            Concentration difference (CLS-NCLS) of oxidized species (M).
-        delta_red : float
-            Concentration difference (CLS-NCLS) of reduced species (M).
+        delta_ox_mols : float
+            Oxidized species crossing at given timestep (mol).
```
Comment (jeremyfell) [inline src/rfbzero/redox_flow_cell.py:400]: `at a given timestep`

File: src/rfbzero/redox_flow_cell.py [hunk 3]
```diff
@@ -391,17 +396,17 @@ def _coulomb_counter(
...
+        delta_red_mols : float
+            Reduced species crossing at given timestep (mol).
```
Comment (jeremyfell) [inline src/rfbzero/redox_flow_cell.py:402]: `at a given timestep`

File: src/rfbzero/redox_flow_cell.py [hunk 4]
```diff
@@ -122,25 +125,26 @@ def __init__(
...
-        self.delta_ox = 0.0
-        self.delta_red = 0.0
+        self.delta_ox_mols = 0.0
+        self.delta_red_mols = 0.0
 
         self.nernst_const = (R * temperature) / F
 
-        for key, value in {'cls_volume': self.cls_volume, 'ncls_volume': self.ncls_volume, 'k_0_cls': self.k_0_cls,
+        for key, value in {'cls_volume': self.cls_volume, 'ncls_volume': self.ncls_volume,
                            'cls_start_c_ox': self.c_ox_cls, 'cls_start_c_red': self.c_red_cls,
```
Comment (jeremyfell) [inline src/rfbzero/redox_flow_cell.py:141]: Thoughts on renaming `cls_start_c_ox` to `c_ox_cls`?

File: src/rfbzero/experiment.py [hunk 5]
```diff
@@ -600,7 +613,7 @@ class ConstantCurrent(CyclingProtocol):
     voltage_limit_discharge : float
         Voltage below which cell will switch to charge (V).
     current : float
-        Instantaneous current flowing (A).
+        Current (A) value used for charging. Negative of this value used for discharging current.
```
Comment (jeremyfell) [inline src/rfbzero/experiment.py:616]: `The negative of this value is used for discharging current.`

**Review by jeremyfell — COMMENTED** [review]
Thanks, got a few minor comments

File: src/rfbzero/redox_flow_cell.py (hunk 1 above)
Comment (ericfell) [reply src/rfbzero/redox_flow_cell.py:161]: forgot to do this, good catch.

File: src/rfbzero/redox_flow_cell.py (hunk 4 above)
Comment (ericfell) [reply src/rfbzero/redox_flow_cell.py:141]: done

File: src/rfbzero/redox_flow_cell.py (hunk 1 above)
Comment (ericfell) [reply src/rfbzero/redox_flow_cell.py:161]: actually for this the value is only allowed to be an integer. User could input say 1.5 and all calculations would work, but it is physically impossible in real world.

**Pull request merged**